index.js

```diff
@@ -122,9 +122,20 @@ function createMainWindow() {
 		autoHideMenuBar: config.get("options.hideMenu"),
 	});
 	remote.enable(win.webContents);
+
 	if (windowPosition) {
 		const { x, y } = windowPosition;
-		win.setPosition(x, y);
+		const winSize = win.getSize();
+		const displaySize = electron.screen.getDisplayNearestPoint(windowPosition).bounds;
+		if((x + winSize[0] < displaySize.x - 8 || x - winSize[0] > displaySize.x + displaySize.width) ||
+		   (y < displaySize.y - 8 || y > displaySize.y + displaySize.height)) {
```

Comment:
Shouldn't it be `y + winSize[1]` similar to `x + winSize[0]`? (+  nit:the condition only contains `or` operators so inner parentheses can be omitted)

Reply:
if y < displaySize.y then the titlebar is out of the screen and the app can't be controlled

y + winSize[1] would refer to the bottom corner, which we shouldn't care if its offscreen
we only care if the top corner is out of the screen (just y value)

also the parentheses could be indeed omitted, if you want